spl/src/memo.rs

```diff
@@ -0,0 +1,29 @@
+use anchor_lang::solana_program::pubkey::Pubkey;
+use anchor_lang::Result;
+use anchor_lang::{context::CpiContext, Accounts};
+
+pub use spl_memo;
+pub use spl_memo::ID;
+
+pub fn build_memo<'info>(ctx: CpiContext<'_, '_, '_, 'info, BuildMemo>, memo: &[u8]) -> Result<()> {
```

Comment:
The memo program always deserializes the bytes into a string, so we _could_ add a little extra dev-ex sugar and have this take a string slice, but not a big deal imo.

https://github.com/solana-labs/solana-program-library/blob/d68329c4b43153bee7f002157b7a16bd1b497f27/memo/program/src/processor.rs#L28-L31

Reply:
@buffalojoec I agreed, I was just concerned about the case where some developers did really wanted to use &[u8] instead of String just in case. Just wanted to cover broader use cases as much as possible. :sunglasses:

Reply:
Agreed, that's why I think it's up to you/Acheron how you want the devs to use it!